AirCasting/SessionViews/SessionCardView.swift

```diff
@@ -62,7 +62,11 @@ struct SessionCartView: View {
     }
     
     var body: some View {
-        sessionCard
+        if session.isInStandaloneMode {
+            standaloneSessionCard
+        } else {
+            sessionCard
+        }
```

Comment:
Maybe one-liner `session.isInStandaloneMode ? standaloneSessionCard : sessionCard

Reply:
I think you can't do that inside `ViewBuilders`. I still do those in one line tho:
```Swift
if session.isInStandaloneMode { standaloneSessionCard } else { sessionCard }
```